docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -173,12 +180,21 @@
     }
 
     &:hover:not(.is-active) {
```

Comment:
I think you need this on a focus state too -- when I click the link I see a grey background for a moment. Same thing for secondary nav